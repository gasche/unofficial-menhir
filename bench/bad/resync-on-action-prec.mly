%token A B C
%%
bof:
  erreur1: { action } %prec A
| erreur2: { action }

baz:
    quux {}

