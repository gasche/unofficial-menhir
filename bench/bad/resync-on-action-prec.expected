File "resync-on-action-prec.mly", line 4, characters 9-10:
Syntax error inside a production.
File "resync-on-action-prec.mly", line 5, characters 9-10:
Syntax error inside a production.
